src/itemset.js

```diff
@@ -96,6 +97,12 @@ define([
                     return value;
                 },
                 validationFunc: function (mug) {
+                    if (!mug.options.lookupTablesEnabled) {
+                        return "You no longer have access to Lookup Tables in your application. " +
+                            "Lookup Tables are available on the Standard plan and higher.\n" +
+                            "Before you can make a new version of your application, " +
+                            "you must [change your subscription](" + changeSubscriptionLink + ") or delete this question.";
```

Comment:
And to be super paranoid, if there is a link, might want to check that the link exists before including it in the message. Bugs causing us to create "#" links that go nowhere will frustrate people.